apps/laboratory/src/components/Ethers/EthersSendCallsWithPaymasterServiceTest.tsx

```diff
@@ -0,0 +1,152 @@
+  const [isLoading, setLoading] = useState(false)
+
+  const { address, chainId, isConnected } = useWeb3ModalAccount()
+  const { walletProvider } = useWeb3ModalProvider()
+  const toast = useChakraToast()
+
+  const paymasterServiceSupportedChains =
+    address && walletProvider instanceof EthereumProvider
+      ? getCapabilitySupportedChainInfoForEthers(
+          WALLET_CAPABILITY_NAMES.PAYMASTER_SERVICE,
+          walletProvider,
+          address
+        )
+      : []
+  const paymasterServiceSupportedChainNames = paymasterServiceSupportedChains
+    .map(ci => ci.chainName)
+    .join(', ')
+  const currentChainsInfo = paymasterServiceSupportedChains.find(
+    chainInfo => chainInfo.chainId === Number(chainId)
+  )
+  async function onSendCalls() {
+    try {
+      setLoading(true)
+      if (!walletProvider || !address) {
+        throw Error('user is disconnected')
+      }
+      if (!chainId) {
+        throw Error('chain not selected')
+      }
+
+      if (!paymasterServiceUrl) {
+        throw Error('paymasterServiceUrl not set')
+      }
+      const provider = new BrowserProvider(walletProvider, chainId)
+      const amountToSend = parseGwei('0.001').toString(16)
+      const calls = [
+        {
+          to: vitalikEthAddress,
+          value: `0x${amountToSend}`
+        },
+        {
+          to: vitalikEthAddress,
+          data: '0xdeadbeef'
+        }
+      ]
+      const sendCallsParams = {
+        version: '1.0',
+        chainId: `0x${BigInt(chainId).toString(16)}`,
+        from: address,
+        calls,
+        capabilities: {
+          paymasterService: {
+            url: paymasterServiceUrl
+          }
+        }
+      }
+      const batchCallHash = await provider.send(EIP_5792_RPC_METHODS.WALLET_SEND_CALLS, [
+        sendCallsParams
+      ])
+      toast({
+        title: 'SendCalls Success',
+        description: batchCallHash,
+        type: 'success'
+      })
+    } catch {
+      toast({
+        title: 'Error',
+        description: 'Failed to send calls',
+        type: 'error'
+      })
+    } finally {
+      setLoading(false)
+    }
+  }
+
+  function isSendCallsSupported(): boolean {
+    if (walletProvider instanceof EthereumProvider) {
+      return Boolean(
+        walletProvider?.signer?.session?.namespaces?.['eip155']?.methods?.includes(
+          EIP_5792_RPC_METHODS.WALLET_SEND_CALLS
+        )
+      )
+    }
+
+    return false
+  }
+
+  if (!isConnected || !walletProvider || !address) {
+    return (
+      <Text fontSize="md" color="yellow">
+        Wallet not connected
+      </Text>
+    )
+  }
+  if (!isSendCallsSupported()) {
+    return (
+      <Text fontSize="md" color="yellow">
+        Wallet does not support this feature
+      </Text>
+    )
+  }
+  if (paymasterServiceSupportedChains.length === 0) {
+    return (
+      <Text fontSize="md" color="yellow">
+        Account does not support paymaster service feature
```

Comment:
why the account? isn't this related to the supported chains by the wallet? Or account = wallet ? 
If that's the case I'd unify the terminology on the messages

Reply:
capability is based on account address and chain, it can be possible to have other account address supporting this capability for same chain.